src/context-selection/org-unit-selector-bar-item/get-offline-levels-to-load.js

```diff
@@ -15,7 +15,7 @@ export default function getOfflineLevelsToLoad({
 }) {
     return userOrganisationUnits
         .map((userOrgUnit) => {
-            const foundLevel = organisationUnitLevels.find(
+            const foundLevel = organisationUnitLevels?.find(
```

Comment:
This crashed for some reason for me, so I just added this as a saveguard. Not sure if necessary though.